src/components/ArticleController.jsx

```diff
@@ -4,14 +4,19 @@ import FalcorController from 'lib/falcor/FalcorController';
 
 export default class ArticleController extends FalcorController {
   static getFalcorPath(params) {
-    // Format: thegazelle.org/issue/:issueId/:category/:articleSlug
-    return ["data", "articles", params.issueId, params.category, params.articleSlug, ["title", "slug", "html"]];
+    // Format: thegazelle.org/issue/:issueId/:articleCategory/:articleSlug
+    return ["issues", params.issueId, "articles", params.articleCategory, params.articleSlug, ["title", "html"]];
```

Comment:
I think this is wrong. The falcorPath actually has no relationship to the actual URL path, and does not need to be similar at all.

In this case the correct function is:

```
return ["articles", "params.articleSlug", ["title", "html"]]
```